frontend/src/hooks/use-app-refresh.ts

```diff
@@ -1,14 +1,25 @@
-import { useEffect } from 'react';
-import { updateRefreshTimer, useAppDispatch } from '../store';
+import { useCallback, useEffect } from 'react';
+import { updateRefreshTimer, useAppDispatch, useAppSelector } from '../store';
 
 export const useAppRefresh = () => {
   const dispatch = useAppDispatch();
 
+  const { refreshTimer } = useAppSelector(state => state.app);
+
   useEffect(() => {
     const refreshInterval = setInterval(() => {
-      if (!document.hidden) dispatch(updateRefreshTimer());
+      if (!document.hidden) dispatch(updateRefreshTimer({}));
```

Comment:
hm, do you think it would be possible to make this param optional rather than using `{}` ? I see its a payload, so might be tricky... Anyway, this is just a nitpick so don't worry if this can't be changed easily.